src/core/lib/security/authorization/rbac_translator.cc

```diff
@@ -337,45 +364,51 @@ absl::StatusOr<RbacPolicies> GenerateRbacPolicies(
     return absl::InvalidArgumentError(
         "SDK authorization policy is not an object.");
   }
-  auto it = json->mutable_object()->find("name");
-  if (it == json->mutable_object()->end()) {
+  auto it = json->object_value().find("name");
+  if (it == json->object_value().end()) {
     return absl::InvalidArgumentError("\"name\" field is not present.");
   }
   if (it->second.type() != Json::Type::STRING) {
     return absl::InvalidArgumentError("\"name\" is not a string.");
   }
   absl::string_view name = it->second.string_value();
-  RbacPolicies rbac_policies;
-  it = json->mutable_object()->find("deny_rules");
-  if (it != json->mutable_object()->end()) {
-    if (it->second.type() != Json::Type::ARRAY) {
-      return absl::InvalidArgumentError("\"deny_rules\" is not an array.");
-    }
-    auto deny_policy_or = ParseDenyRulesArray(it->second, name);
-    if (!deny_policy_or.ok()) {
-      return absl::Status(
-          deny_policy_or.status().code(),
-          absl::StrCat("deny_", deny_policy_or.status().message()));
+  RbacPolicies rbacs;
+  std::unique_ptr<Rbac> allow_rbac;
```

Comment:
Same comment as above: This could use `absl::optional<>` instead.

Or better yet, this could just be a bool indicating whether the field was seen, and if it is seen, you can directly set `rbacs.allow_policy`, so you don't need to move it later.

Reply:
Done